### PR TITLE
Adapt angular: ngeo.query

### PR DESCRIPTION
**Pull request by pfirpfel**

### CODE REVIEW

File: src/query/Querent.js [hunk 1]
```diff
@@ -1,17 +1,23 @@
-goog.provide('ngeo.Querent');
+/**
+ * @module ngeo query namespace
```
Comment (ger-benjamin) [inline src/query/Querent.js]: what's that ? Do you know ?

File: src/query/bboxquerydirective.js [hunk 2]
```diff
@@ -1,8 +1,13 @@
-goog.provide('ngeo.bboxQueryDirective');
+/**
+ * @module ngeo query namespace
+ */
+goog.provide('ngeo.query.bboxQueryDirective');
```
Comment (ger-benjamin) [inline src/query/bboxquerydirective.js]: Event if that's always an angular directive, can you names it "component ?".
That's a more generic term and In the future, we will change it to a component. So we (and our customers in their custom code) will have to update again the require.

File: contribs/gmf/src/services/querymanager.js [hunk 3]
```diff
@@ -3,7 +3,7 @@ goog.provide('gmf.QueryManager');
 goog.require('ol.events');
 goog.require('gmf');
 goog.require('gmf.Themes');
-goog.require('ngeo.Query');
+goog.require('ngeo.query.Query');
```
Comment (ger-benjamin) [inline contribs/gmf/src/services/querymanager.js]: ngeo.query.Service ?  query Query looks weird (In my own opinion)

Comment (gberaudo) [reply contribs/gmf/src/services/querymanager.js]: Yes, should be Service.

File: src/query/Querent.js (hunk 1 above)
Comment (gberaudo) [reply src/query/Querent.js]: We should not put `@module` annotations; they will be generated automatically later (in a specific format).

File: src/query/bboxquerydirective.js (hunk 2 above)
Comment (gberaudo) [reply src/query/bboxquerydirective.js]: +1

File: src/query/mapquerydirective.js [hunk 4]
```diff
@@ -119,4 +123,11 @@ ngeo.mapQueryDirective = function(ngeoMapQuerent, $injector) {
...
 
-ngeo.module.directive('ngeoMapQuery', ngeo.mapQueryDirective);
+
+ngeo.query.mapQueryDirective.module = angular.module('ngeoMapQuery', [
+  ngeo.query.MapQuerent.module.name,
+]);
+
+ngeo.module.requires.push(ngeo.query.mapQueryDirective.module.name);
+
+ngeo.query.mapQueryDirective.module.directive('ngeoMapQuery', ngeo.query.mapQueryDirective);
```
Comment (gberaudo) [inline src/query/mapquerydirective.js]: I think we said to export the Angular module for directives and components (not the directive function).
```
ngeo.query.mapQueryDirective = angular.module('ngeoMapQuery', [
  ngeo.query.MapQuerent.name,
]);
```

Comment (ger-benjamin) [reply src/query/mapquerydirective.js]: +1
This file is strange. That should be:
`ngeo.query.mapQueryDirective = angular.module...`
`ngeo.module.requires....`
...
`ngeo.query.mapQueryDirective.directive_ = function...`
...
`ngeo.query.mapQueryDirective.directive('ngeoMapQuery', ngeo.query.mapQueryDirective.directive_);`

no ?

**Pull request merged**